<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Form</class>
  <widget class="QWidget" name="Form">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="maximumSize">
      <size>
        <width>800</width>
        <height>600</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Steam Engine Pac-Man</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
        <rect>
          <x>60</x>
          <y>40</y>
          <width>661</width>
          <height>511</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="playButton">
            <property name="text">
              <string>Play</string>
            </property>
            <property name="flat">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="multiplayerButton">
            <property name="text">
              <string>Multiplayer</string>
            </property>
          </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="selectLevelButton">
            <property name="text">
              <string>Select Level</string>
            </property>
            <property name="flat">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="replayButton">
            <property name="text">
              <string>Replay</string>
            </property>
          </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="helpButton">
            <property name="text">
              <string>Help</string>
            </property>
          </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="exitButton">
            <property name="text">
              <string>Exit</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources />
  <connections />
</ui>
